### PR TITLE
:sparkles: sticker aliases

### PR DESCRIPTION
**Pull request by nothingelsematters**
do we really need this fucking feature?(  'cuz i've tried so hard and got so far...

### CODE REVIEW

File: src/main/kotlin/com/github/pool_party/pull_party_bot/command/Messages.kt [hunk 1]
```diff
@@ -10,6 +10,8 @@ package com.github.pool_party.pull_party_bot.commands
 // /help command suggestion
 // """
 
+// TODO messages for alias commands
```
Comment (ShuffleZZZ) [inline src/main/kotlin/com/github/pool_party/pull_party_bot/command/Messages.kt:13]: I believe it would be nice to have messages before the merge

Comment (nothingelsematters) [reply src/main/kotlin/com/github/pool_party/pull_party_bot/command/Messages.kt:13]: I would let **you** do this task, u r the best in  this field. And you can look at the command interface, as well. U can make more revisions on some kind of user experience, idk.

Btw, I am going to fix linter & code factor warnings later this evening

**Comment by ShuffleZZZ** [comment]
Friendly messages waiting room, I guess

**Review by ShuffleZZZ — CHANGES_REQUESTED** [review]
AliasTransactiomns.kt doesn't seem like a correct file name

File: src/main/kotlin/com/github/pool_party/pull_party_bot/database/transaction/Transactions.kt [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+fun initDB() { // change token for another app
+    Database.connect(
+        System.getenv("JDBC_DATABASE_URL"),
+        user = System.getenv("JDBC_DATABASE_USERNAME"),
+        password = System.getenv("JDBC_DATABASE_PASSWORD")
+    )
+
+    transaction {
+        addLogger(StdOutSqlLogger)
+        SchemaUtils.create(Chats, Parties)
```
Comment (ShuffleZZZ) [inline src/main/kotlin/com/github/pool_party/pull_party_bot/database/transaction/Transactions.kt:22]: Is it intended to try to create those tables twice?

**Review by ShuffleZZZ — COMMENTED** [review]
I'm done with languishing this project, we can roll it out to develop under your responsibility

**Comment by nothingelsematters** [comment]
> under your responsibility

And here we go again. You ain't wanna do anything, as usual 😒

**Comment by ShuffleZZZ** [comment]
> > under your responsibility
> 
> And here we go again. You ain't wanna do anything, as usual 😒

You naigralsya, no approve any time soon

**Comment by nothingelsematters** [comment]
@ShuffleZZZ cmon, bruh, just check it, u sick fuck

**Review by ShuffleZZZ — APPROVED** [review]
The decision I will always regret

**Review by ShuffleZZZ — CHANGES_REQUESTED** [review]
Ok, c ya

**Pull request closed without merging**